### PR TITLE
feat(BBD-825): create cart

### PR DESCRIPTION
**Pull request by treeforever**

### CODE REVIEW

File: src/core/index.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import Configuration from './configuration';
 import ProductTransformer from './product-transformer';
```
Comment (Vic-Dev) [inline src/core/index.ts]: do `import ProductTransformer, { TransformUtils } from './product-transformer';`

File: src/core/index.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
...
+import { TransformUtils } from './product-transformer'
```
Comment (Vic-Dev) [inline src/core/index.ts]: Remove

File: src/services/cart.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import { core } from '../core/service';
+import LazyService from '../core/service/lazy';
+import Tag from '../core/tag';
+import { WINDOW } from '../core/utils';
+import StoreFront from '../storefront';
+
+@core
+class CartService extends LazyService {
+  lazyInit() {
+    // this.app.flux.on(Events.ADD_TO_CART, this.addToCart);
```
Comment (Vic-Dev) [inline src/services/cart.ts]: Don't forget to do this!

**Review by Vic-Dev — CHANGES_REQUESTED** [review]
Small things :) It's been a 🌲-t working with you!

**Comment by jmbtutor** [comment]
Archiving repo. Closing.

**Pull request closed without merging**